test/core/client_channel/lb_policies_test.c

```diff
@@ -780,15 +783,21 @@ static void verify_total_carnage_round_robin(const servers_fixture *f,
     }
   }
 
-  /* no server is ever available. The persistent state is TRANSIENT_FAILURE */
+  /* no server is ever available. The persistent state is TRANSIENT_FAILURE. May
+   * also be CONNECTING if, under load, this check took too long to run and some
+   * subchannel already transitioned to retrying. */
   for (size_t i = 0; i < sequences->n; i++) {
     const grpc_connectivity_state actual = sequences->connectivity_states[i];
-    const grpc_connectivity_state expected = GRPC_CHANNEL_TRANSIENT_FAILURE;
-    if (actual != expected) {
+    uint32_t expected_bitset;
```

Comment:
The bitsets really seem like overkill here.  Why not just say `if (actual != GRPC_CHANNEL_TRANSIENT_FAILURE && actual != GRPC_CHANNEL_CONNECTING)`?  I think it's both shorter and easier to understand.